app/components/candidate_interface/gcse_qualification_review_component.rb

```diff
@@ -65,14 +65,19 @@ def grade_row
     end
 
     def present_grades
-      if application_qualification.subject == ApplicationQualification::SCIENCE_TRIPLE_AWARD
+      case application_qualification.subject
+      when ApplicationQualification::SCIENCE_TRIPLE_AWARD
         grades = application_qualification.structured_grades
         [
           "#{grades['biology']} (Biology)",
           "#{grades['chemistry']} (Chemistry)",
           "#{grades['physics']} (Physics)",
         ]
-      elsif application_qualification.subject == 'english' && application_qualification.structured_grades
+      when ApplicationQualification::SCIENCE_DOUBLE_AWARD
+        "#{application_qualification.grade} (Double award)"
+      when ApplicationQualification::SCIENCE_SINGLE_AWARD
+        "#{application_qualification.grade} (Single award)"
+      when ->(subject) { subject == 'english' && application_qualification.structured_grades }
```

Comment:
Interesting, did not know you could use lambdas in case statements 👍

Reply:
The reason I needed this was because it was the only way I could find to allow an AND statement. I don't think it's particularly readable, so please shout if there's a more standard way.

Reply:
+1